Objective-C/CBLListenerToken.h

```diff
@@ -23,7 +23,7 @@ NS_ASSUME_NONNULL_BEGIN
  Listener token returned when adding a change listener. The token is used
  for removing the added change listener.
  */
-@protocol CBLListenerToken <NSObject>
+@protocol CBLListenerToken <NSObject, NSCopying>
```

Comment:
Ugh, I don't think we should NSCopying as this means that our API allows the token to be copied :(. So reverting back to use UUID  as a key is better. Sorry for this.